Lesson7v3/auth.py

```diff
@@ -0,0 +1,74 @@
+import json
+
+with open("db.json", "r") as f_json:
+    data = json.load(f_json)
+
+
+def check_name(data):
+    """Here we are getting a nickname for a new users registration and checking it for individuality"""
+    active = True
+    while active:
+        username = input('Enter your nickname for a registration.' "\n")
+        if username in data.keys():
+            print("This nickname is already taken")
+        elif 10 < len(username) or 2 > len(username):
+            print("The name must consists of less than 10 chars and more than 1")
+        else:
+            return username
+
+
+def get_password():
+    password = input('Create your password'"\n")
+    while input("Repeat your password:"'\n') != password:
```

Comment:
here looks like we should repeat only the second password? better to include the first password to repeat it also..
what will we do if our system don't show us our passwords and we forgot the first